packages/origin-ui-core/webpack/base.config.js

```diff
@@ -71,5 +71,9 @@ module.exports = {
 
     node: {
         fs: 'empty'
-    }
+    },
+
+    externals:[{
+        xmlhttprequest: '{XMLHttpRequest:XMLHttpRequest}'
```

Comment:
Why do we need that? : /

Reply:
TBH can't remember. There was some error and this fixed it. Will check again.